audit/projects/k8s-infra-e2e-boskos-001/services/compute/project-info.json

```diff
@@ -3,7 +3,7 @@
DAQABAAABAQDIm/Z3K0oPSZa0eLXTydSTcJFy9Gj1bE5QdAJt61f6abW0SQqlcez+PScnQFyqU8AFFRtO4kXB0JyDcobF3qQSXTN3P/NvV2psw/lHBes7uScfVDvfm7nDK1ndEbb24wBzXdc4wdZeCW+NoDEa8btslsSoGgINsUeI/oyME872WalXQuSiIFy4R7P3XmCXSePb8b+4HUa7vd5IpB+2K18oTmf+F93ZSPRJxTk8ZCJME6LnN7LxJkHYpnO+hC9IMPbAzWc0LDWpCNDuu0LO2rtvP/y/opj4it6i8l+FUg6hAHPlUos6vd3DCjkf3ylBwCAMOdRJp27DJmLx8+U+jiRz root@1e8e278f-f707-11ea-ae6d-8ed64f3416c0\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDpcmjugW/itfeCL7WeZnea2YOEfzZgxMPTsvuHk39woyTRxJdjq4v/zMLSOXNBrdZFyqdW+F2ySS2GxCpvR5O2QKxEiqCcGOPK3xdQNRIP5mHtELgoPvPc4i22u3+ipfB/CrdrjN/ELwpUZXHrah5bNxOivEbNwYvQ3bMq+WbduRRLdKr10fwdyErywnqex4PvolxR2bOAnNdhakoUH31pSSECBKOX6YUh+TOG2Hh4wpyAJxSwxg7o5IgglU+ok9i6lK8g42CSlq+NBRp2AmoXd82KzXBfqpbTMJd2A8EnZrtq/VKGXFpWE4BzlA6+H7y3jxfcvTDfxH7I7YTgRUvB root@d99b53dd-f8b6-11ea-926e-7e7ea190b727\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDtu4A2VuzGX/fH0HUjKcVqBi/+Fdd/kigCvQMWF3Uw6k8Q71l7ewcDDtfs+WDk06bNg50JsdaA+OcetrpL0yLPuKBvtSg/vl8msn8uxPbhHAgQ/QaaTLWYnztioPEsX4GHj6IcskwLGWNR+mIlqxY+cximmQ1O7hj1IGPNLuePysM9ZdpMDT7xOVc73PXd/t5+kZPjnKEQlsz5Zd7FtQH6QJ7ptKLYfcS062ZQlQJNaQPVIPk/TA9xLAEHaTSw0u1eJHuyXvbSbqvj8e/69wWMR32QmN6mKus3hQZjPMm+DzmDIWq2wt96i1eickCGSvpfXG4j6TIYmTVu3yTDe+ER root@1d6a8847-0838-11eb-b895-6272783fc925\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC5eYrnUmX0URxEg778N+rOLSPqYJk1nPe3ppy/X7zf6rs514UlmBpftsI+ZC5iFxJKIfI1+IECBydUmMtcMeDNq6TJ+QtPaQjonX0jp7Cm4fdOmTfiDnuY1qhepvdKN7oWmZeNT1xrdAe8qeeLIbup92zIlU8++nusMmrAyvmmSjPtl41YNufj3g+4MEryQ6lq43RdkXT35GfZPdWUD3ZUrsc2b568NzBCdCG0j+/RYWHUxnRWiReJzhcrZX+dFACNL7Br5UVz6vmWNV8hLfrVJAAjHNtqWP0m4VCUOX84dfvmTa5I1xOjrqMNMLdWhNCKbR8zOJfaM86lzh66+yQv root@91bc69a5-532d-11eb-b6a5-7e8b7adfd54c\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC2cYpeZLD9y25W19mQbvRxg7slzivZIV8y+p8XCIT2xddSl1HQ7Fz1Ir1iUzq0cqniTzHnmM8+RC47XaMMOaXaZjjPcpyNiaFOczCn9c4M5E48ipAa105HP9vj1yewWfgCbPxRP6QSweR8BOVLZHUpZwebPen1XwXUYrUBC0rhPHcFKPUtla4hlUEqI9/ca1yiohCUWDSpO9y5wazA54rZhkN/AHxhPCE0v8xcjJKbcZoUdyFFjY7kMXsL6AOpNwqpFzmhi01+ei3BikcRyKQnWAK15n+F0N4d1tYW5FPEAeepx/BsUnAlX3pji6N11c+sPg3laspWOvVTFgBWrpDd root@12716507-5573-11eb-b690-769430c9414e\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1HW5T8W/GHe9ba3zZhCR59IjYskzVSoR9KQQF97qG37mPsgvp+HZaFN4czh5Nq5SL7EbCZsROj50cMaO4Q37wCKvIQoWQtaFnBZkzvcADqPAyvIkGjsVRP7+3bSW+2vFr1bqmE8X5lFvIrAdR6BFbgirfTdeOT4SZo6OzyXm4mznYFQMxz5MIqNA+64qMq7UKWKYULRI33YMn8kvUVi3/sATPjOt4v0lQAM8i0g6IKw9MYlLNBQ9G7nH3tYWBBXAMboOT5TsGKGSt8FooNhxKuygbSyJoQL9T5x24sDjxqOVMvtTLRk3NtigdIcigiFgbRxGSmoNXGg9OIIP5Bcyh root@df1a6954-82ae-11eb-b915-56a8d6cedae4\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDfySXJDe/9NLf9ek6OEu5CKV4ugzXnI4I+Ealg9fqaB2+S0BCsSGnrCOGUjKH0PWy85VqwhIhd+rTPz3sZRsbWTVvtnTWgLzBFnC20TuJlhmj+EUIIJcigSrkQcOOpGJi2trDBr1eXAN6SY9ZqHQRicndDVsJ9oi9eXrTUYoSwM+gbIWxscrRdvnsT61m1wLSmxJOoEb9013ow66j+RhDRd1MtJrrF1qoKOIPo9FtVHoOJuZ8gV2mASemGGx1xiTZSboDubvkKVroGzhG+uVEXaTqh7GJsmdN8yvjw/9qfphU7ihDuqZCIpiFx3HNV4tYsA6h1DhD92E7mInGC7Zsn root@d594c422-84c1-11eb-b098-eeb808a18dc8"
```

Comment:
All of these are happening due to https://github.com/kubernetes/k8s.io/issues/1751#issuecomment-790171952

Need to implement https://github.com/kubernetes-sigs/kubetest2/issues/109#issuecomment-797332161 to quiet down this noise 

Then consider purging ssh keys back to a known state, ref: https://github.com/kubernetes/k8s.io/issues/1751